index.html

```diff
@@ -427,6 +427,8 @@ <h2>Glossary</h2>
 
 <p><dfn class="lint-ignore export">Spacing mark</dfn>. Combining characters that consume space along the baseline, either before or after the base character.</p>
 
+<p><dfn class="link-ignore export">Spillover</dfn>. Errors in text presentation due to improper [=bidi isolation=]. When one string is inserted into another string, unless the inserted value is isolated and given the proper [=base direction=], the [=bidi algorithm=] can visually rearrange the text in unexpected ways that make the text illegible. For example, a MessageFormat pattern string like <code lang="en" translate="no">{{$restaurantName} - {$numReviews} reviews}</code> which inserts a <kbd>numReviews</kbd> of <kbd>4</kbd> and a <kbd>restaurantName</kbd> of <kbd lang="he" translate="no">&#x5e4;&#x5d9;&#x5e6;&#x5d4; &#x5e1;&#x5d2;&#x5d5;&#x5dc;&#x5d4;</kbd> (which means roughly <em>Purple Pizza</em> in Hebrew) would be presented as <code lang="en" dir="ltr">‎פיצה סגולה - 4 reviews</code>. The change in text order is due to spillover. Adding markup or isolating controls around the inserted values fixes the result: <code><span lang="he" translate="no" dir="rtl">&#x5e4;&#x5d9;&#x5e6;&#x5d4; &#x5e1;&#x5d2;&#x5d5;&#x5dc;&#x5d4;</span> - <span dir="ltr">4</span> reviews</code></p>
```

Comment:
due to improper [=bidi isolation=] -> due to a lack of [=bidi isolation=]

i don't think you need to show the (lengthy) source code: just show what it should look like and what it turns out as.  That would enable you to show a second example based on list ordering.

Reply:
We should point to https://www.w3.org/International/articles/inline-bidi-markup/index.en.html for more information.